test/testSetup.ts

```diff
@@ -165,6 +185,83 @@ export class TestSetup {
     utilities.log('');
   }
 
+  public async verifyAllExtensionsAreRunning(): Promise<void> {
```

Comment:
A new function that makes sure all Salesforce extensions are present and activated.  Called at the beginning of the workflow during setup and every time the E2E test runs "Developer: Reload Window".

Reply:
Can we move this to extensionUtils? seems like a better place to be as it's going to be used in anInitialSuite too

Reply:
It's moved now!